addons/web/tests/test_image.py

```diff
@@ -168,7 +168,7 @@ def get_datetime_from_token(token):
             return datetime.fromtimestamp(int(token.rsplit("o", 1)[1], 16))
 
         def get_datetime_from_record_field(record, field):
-            return get_datetime_from_token(limited_field_access_token(record, field))
+            return get_datetime_from_token(record._get_raw_access_token())
```

Comment:
This makes the getter only working for attachment and no longer for any record, and the field param is ignored?

If this is what we want, the getter can be removed now. If we have other usage, it should be adapted.